lib/src/github/models.rs

```diff
@@ -96,6 +96,40 @@ pub struct App {
     pub name: String,
 }
 
+#[derive(Deserialize, Serialize, Clone, Debug)]
+pub struct Team {
+    pub name: String,
+    pub slug: String,
+    pub html_url: String,
+}
+
+impl Team {
+    pub fn new(org: &str, slug: &str) -> Team {
+        Team {
+            name: slug.to_string(),
+            slug: slug.to_string(),
+            html_url: format!(
+                "https://github.com/orgs/{}/teams/{}",
+                org.to_string(),
+                slug.to_string()
```

Comment:
```suggestion
                org,
                slug
```

per the warnings here: you should just be able to pass the `&str` directly in `format!` -- no need to convert them to a String first.